go-controller/pkg/clustermanager/egressservice/egressservice_cluster.go

```diff
@@ -324,6 +330,13 @@ func (c *Controller) repair() error {
 			klog.Errorf("Failed to read EgressService key: %v", err)
 			continue
 		}
+		if es.Spec.SourceIPBy != egressserviceapi.SourceIPLoadBalancer {
+			err := c.setEgressServiceHost(es.Namespace, es.Name, noSNATHost)
```

Comment:
nope, because I like avoiding nested ifs (:sweat_smile:) and I think it won't do anything if it's already the desired status. do you prefer I handle it anyways (with a nested if or another conditional)?